uberfire-workbench/uberfire-workbench-client-views-patternfly/pom.xml

```diff
@@ -574,13 +576,20 @@
                     <include>dev/vs/basic-languages/tcl/tcl.contribution.js</include>
                     <include>dev/vs/basic-languages/twig/twig.contribution.js</include>
                     <include>dev/vs/basic-languages/typescript/typescript.contribution.js</include>
+                    <include>dev/vs/basic-languages/typescript/typescript.js</include>
                     <include>dev/vs/basic-languages/vb/vb.contribution.js</include>
                     <include>dev/vs/basic-languages/xml/xml.contribution.js</include>
                     <include>dev/vs/basic-languages/yaml/yaml.contribution.js</include>
+                    <include>dev/vs/language/typescript/tsMode.js</include>
+                    <include>dev/vs/language/typescript/tsWorker.js</include>
+                    <include>dev/vs/language/typescript/languageFeatures.js</include>
+                    <include>dev/vs/language/typescript/workerManager.js</include>
+                    <include>dev/vs/language/typescript/lib/typescriptServicesMetadata.js</include>
+                    <include>dev/vs/language/typescript/lib/typescriptServices.js</include>
```

Comment:
hi @romartin this class has 7.8mb. Is this really necessary?

Reply:
Also, it would be nice to review if there is another approach then add all those libraries. 
Unpacked DMN editors before this PR: 33.9Mb
Unpacked DMN editors after this PR: 42.9Mb

Reply:
hey @ederign 

Well honestly I cound't make javascript language working property without each one of those added files, it was comtinously complaining and I was adding one by one. On the other hand, it results on typescript also supported, which was not the goal for this task, but the editor doens't work for javascript without those libraries, I tested everything I could... So I think those should be there. yes.

On the other hand:
- We're relying on not minimized packages for the monaco editor, as the production ones are not working properly (thanks to @karreiro for pointing this), so we could try to improve here when some newer library version appears.
- We're adding this libraries for all editors, which is not efficient, agree. 

I can try to play around with POM's and modules and see if I can just include those libraries for the BPMN editor, not at platform level, but honestly don't know yet if it's possible or how tricky it results, will let you know. WDYT? Did you already tested some similar approach?

Thanks guys!

Reply:
Hi @romartin, I understand the issue, and it's not simple indeed :-( As an alternative, we could try to mock the module defined by this file. Wdyt? :-)

Reply:
what you mean by "mocking" it? :) I'm open for ideas yeah!! :+1:

Reply:
As far as I know those javascript modules are not being loaded if not needed, so only BPMN will really load that modules. But it's true that the size for all editors will increase, even if those js modules are not being loaded.

Reply:
@romartin I'm wondering if we could just have something like this (instead of adding `typescriptServices.js`):

```javascript
define("vs/language/typescript/lib/typescriptServices", [], function() { return { /* maybe we'll need a fake API implementation here */ }; });
```

Thus, we would keep Monaco dependency tree happy and avoid this extra 7mb (if this file is not used at runtime).